js/main.js

```diff
@@ -229,11 +229,16 @@ $(window).bind('keydown', function(e) {
         // Select all text in address bar
         $('.js-navAddressBar').select();
         break;
+      case config.keyShortcuts.save:
+        // Trigger any visible .saveForm button
+        // .last() will make sure you'll only click one button if 2 are visible (listing form)
+        $('.js-saveForm:visible').last().trigger('click');
```

Comment:
I don't think we want to depend on a class for this, or assume the only way to save a form is by clicking a button. If there's a form that saves through some other means, this won't work.

It's also entirely possible save buttons may not be visible even though a form is in a saveable state, due to layout tricks and effects.

Instead, fire an event on window.obEventBus, like this:

window.obEventBus.trigger('saveCurrentForm);

Then, in each view that has a saveable form, listen for that event, and trigger the view's save function when the event happens.